wasmegg/artifact-explorer/update-loot-data.sh

```
@@ -9,6 +9,6 @@ echo "HTTP ${HTTP_CODE}"
 if [ -s loot_raw.json ]; then
   echo 'updating loot.json'
   # jq instead of mv to get rid of evil windows newlines
-  jq -r . loot_raw.json > "$(dirname $0)"/src/lib/loot.json
+  jq -r . loot_raw.json > "$(dirname $0)"/data/loot.json
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)